src/aind_data_access_api/document_db.py

```diff
@@ -28,13 +28,15 @@ def __init__(
         collection: str,
         version: str = "v1",
         boto_session=None,
+        use_signed_requests=False,
```

Comment:
Are any current services using this client to write to docdb? If so, this would be a breaking change because they would now need to set this option as True.